dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -15,16 +15,17 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
   setup do
     # place in setup instead of setup_all otherwise course ends up being serialized
     # to a file if levelbuilder_mode is true
-    @unit_group = create(:unit_group)
+    @unit_group = create(:unit_group, published_state: SharedConstants::PUBLISHED_STATE.beta)
```

Comment:
I like that the validation forces you to specify a value even though there is a database default.